docker/child.conf

```diff
@@ -0,0 +1,135 @@
+RemoteServer unrealircd
+Port 6999
+
+# Set to 1 to use secure connections (must be compiled with --enable-gnutls)
+SSL 0
+
+# The IP to bind to connect to the remote server and for the partyline
+# If you have only one IP, you don't need to change this setting.
+# Examples :
+# LocalAddr 127.0.0.1
+# LocalAddr ::ffff:127.0.0.1
+# LocalAddr ::1
+
+# Link pass
+LinkPass "LiNk"
+
+# Modules to load
+# Normally you want to load all these modules
+LoadMod "m_help m_bot m_chan m_host m_nick m_oper"
+# You can load additionnal modules, such as secserv, spamscan, or connect4
+LoadMod "secserv"
+
+# The SetCmdLev option allows you to change the level of a command. The syntax is "command newlevel" where command is what you /msg C. You can let the example given if you don't want to enable the botserv feature for users (level 100 required to use the commands). Also, please note that if you don't change the level of the help for the command, it will be listed anyway.
+
+SetCmdLev "chan assign 100"
+SetCmdLev "chan unassign 100"
+SetCmdLev "chan botlist 100"
+SetCmdLev "help chan assign 100"
+SetCmdLev "help chan unassign 100"
+SetCmdLev "help chan botlist 100"
+
+# The maximum number of clones allowed in the network
+MaxClones 5
+
+# Numbers of days after a unused nick and a unused channel will expire
+NickExpire 120
+ChanExpire 90
+
+# The maximum number of channels a user can register
+MaxChanPerUser 10
+
+# The different access level, you can let the default value
+LevelOper 100
+LevelAdmin 500
+LevelRoot 900
+LevelOwner 1000
+
+# Access levels for channels
+ChanLevSAdmin 20
+ChanLevAdmin 10
+ChanLevOp 5
+ChanLevHalfop 4
+ChanLevVoice 3
+ChanLevInvite 1
+ChanLevNoStatus -1
+ChanLevAKick -2
+ChanLevAKickBan -3
+
+# Mysql host to connect to
+MysqlHost "mysql"
+
+# Mysql database name
+MysqlDB "child"
+
+# Mysql login
+MysqlUser "root"
+
+# Mysql password
+MysqlPass "child"
+
+# Logfile
+LogFile "child.log"
+
+# The interval in minutes between database saving. One hour is a good value.
+SaveDB 60
+
+# The prefix of the nick which is used when a user didn't logged in and the timeout expired. Ex: prefix123456
+GuestPrefix "Geek"
+
+# Set the listen_port setting to a value between 1024 (I hope you won't start the services as root) and 65535 to enable the partyline.
+# You can connect to the partyline via this port. Only those who are level owner can connect to the partyline, or those who have the CanPl flag (must be set in the partyline by an owner with the .canpl command)
+# The partyline allows you to have full control over the services, more than with the usual irc commands.
+# If you don't know what you are doing, DO NOT enable the partyline.
+ListenPort 0
+
+# The logfile for the partyline. Please note that after a user has logged in, all his/her commands are logged and sent to the other users connected to the partyline.
+PartyLineLog partyline.log
+
+# The exec command of the partyline allows you to execute shell commands. Highly not recommended.
+Exec 0
```

Comment:
Exec is deprecated since https://github.com/Ethylix/child/commit/96a3dd9a9829b6b870d89229f76b4cc5d7704cd9